app/controllers/items_controller.rb

```diff
@@ -0,0 +1,45 @@
+class ItemsController < ApplicationController
+
+  def destroy
+    @user = User.find(params[:user_id])
+    @item = @user.items.find(params[:id])
+
+    if @item.destroy
+      flash[:notice] = "The item was obliterated!"
+    else
+      flash[:notice] = "Oops!, the item could not be deleted.  Try again!"
+    end
+
+  respond_to do |format|
+    format.html
+    format.js
+  end
+end
+
+  def create
+    @user = User.find(params[:user_id])
+    @items = @user.items
+
+    @item = Item.new(item_params)
+    @item.user = current_user
```

Comment:
Instead of the two lines here where you create the new item and then set the current user. You can use this single line:

current_user.items.build(item_params)